src/python/pants/backend/project_info/tasks/export_dep_as_jar.py

```diff
@@ -242,7 +266,8 @@ def _process_target(
             # this means 'dependencies'
             "targets": [],
             "source_dependencies_in_classpath": [],
-            "libraries": [],
+            "compile_libraries": [],
+            "runtime_libraries": [],
```

Comment:
Is this going to introduce breaking changes, or is it okay for us to update fastpass atomically as well? cc @olafurpg

Reply:
Chatted with Olaf, this is fine.